### ISSUE TITLE
Test navigation contrast across stylesheets

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-foundation#74]
This issue addresses a parallel problem present in the theme for Omeka S (https://github.com/omeka-s-themes/foundation/issues/83). Test the menu link contrast across all stylesheets for both horizontal and vertical layouts.

**Comment by kimisgold** [comment]
I had missed styling a number of the mobile elements. The latest commits should hopefully address all your points and area ready for a look.

**Comment by aviannamiller** [comment]
I apologize as I missed a few things, so the following has some new points as well as responses to these changes.

### Horizontal dropdown:
- The dropdown arrows on the mobile menu for Inkwell are 1.2:1 and then 1.5:1 for active links.
- For Seafoam, hover over an active child link changes the text color to #c5f4f2, which is a 4.3:1 contrast with the #007a76 background.
- Default and Revolution - the menu toggle button is styled in white, and it’s about a 1.2:1 contrast for both. They do look okay on hover/focus, however.

### Full height column:
- For Default, Revolution, and Seafoam, I can only see the dropdown arrows if they’re on an active page link (or the parent of). Otherwise, they’re 1:1.
- For Inkwell, the dropdown arrow is black and has good contrast, except for an active page link (or the parent of). In that case, the arrow is styled in white and thus has too low of a contrast. Also, the menu toggle button is white, so again too low of a contrast against #fc0. Arrows on the mobile toggled menu are also problematic, styled in #fc0 against the #e6e6e6 background (1.2:1) and white on #fc0 for active page links (1.5:1).

**Comment by kimisgold** [comment]
Another pass is pushed and ready to test.

**Comment by aviannamiller** [comment]
All looks good, thank you! The only remaining issue I see is with the hover style on active child page links on Inkwell, full height column, on the mobile menu only:

https://github.com/user-attachments/assets/070426e9-bc42-4c62-ab3f-2cd324691500

Video description: Detail of vertical mobile menu. Nested page links are expanded, revealing an active child page link indicated by yellow fill. Hovering over menu links reveals a yellow underline, except on the active link, which loses some of the fill color on its left side on hover.

**Comment by kimisgold** [comment]
Just pushed a fix to prevent that border from appearing in the mobile menu on hover or active states.

**Comment by aviannamiller** [comment]
The variation on horizontal dropdown overall seems pretty static on hover, whereas on the full height column one I see this slight (padding?) jump on the left side on active child links when expanding / collapsing the parent (the same occurs on hover):

https://github.com/user-attachments/assets/37456c3f-0243-4179-bca1-5a43958b5f19

Video description: Vertical mobile menu. Nested page links are expanded, revealing an active child page link indicated by yellow fill. Some of the fill color on the left side disappears when the nested links are expanded, the child link is hovered over, and again when the links are collapsed.

**Comment by kimisgold** [comment]
Just removed a leftover pseudo element style causing the issue only on Chrome.

**Comment by aviannamiller** [comment]
That's great, thanks. All set here.
